build/rollup.config.mjs

```diff
@@ -61,7 +61,7 @@ const rollupConfig = {
 }
 
 if (!ESM) {
-  rollupConfig.output.name = 'arizona-bootstrap'
+  rollupConfig.output.name = 'arizonaBootstrap'
```

Comment:
I guess another alternative to this would be to just use `bootstrap` here and not update the docs JS files...

Not sure how likely it is that custom site implementations of Arizona Bootstrap would need to reference this global object variable name...